core/src/kmx/kmx_environment.cpp

```diff
@@ -20,31 +20,31 @@ namespace {
 }
 
 KMX_Environment::KMX_Environment() {
-  Set(KM_KBP_KMX_ENV_PLATFORM, DEFAULT_PLATFORM);
-  Set(KM_KBP_KMX_ENV_BASELAYOUT, DEFAULT_BASELAYOUT);
-  Set(KM_KBP_KMX_ENV_BASELAYOUTALT, DEFAULT_BASELAYOUTALT);
-  Set(KM_KBP_KMX_ENV_SIMULATEALTGR, DEFAULT_SIMULATEALTGR);
-  Set(KM_KBP_KMX_ENV_BASELAYOUTGIVESCTRLRALTFORRALT, DEFAULT_BASELAYOUTGIVESCTRLRALTFORRALT);
+  Set(KM_CORE_KMX_ENV_PLATFORM, DEFAULT_PLATFORM);
+  Set(KM_CORE_KMX_ENV_BASELAYOUT, DEFAULT_BASELAYOUT);
+  Set(KM_CORE_KMX_ENV_BASELAYOUTALT, DEFAULT_BASELAYOUTALT);
+  Set(KM_CORE_KMX_ENV_SIMULATEALTGR, DEFAULT_SIMULATEALTGR);
+  Set(KM_CORE_KMX_ENV_BASELAYOUTGIVESCTRLRALTFORRALT, DEFAULT_BASELAYOUTGIVESCTRLRALTFORRALT);
```

Comment:
We could trim these down too (maybe seperate PR) `KM_CORE_KMX_ENV_PLATFORM` =>  `KM_CORE_KMX_PLATFORM`
or even drop the `KMX_` also

Reply:
otherwise lgtm

Reply:
We could ... but the longer name clarifies that these are really only used by the KMX environment at present.